packages/terra-table/docs/guides/SectionWithinSection.md

```diff
@@ -0,0 +1,144 @@
+# Terra Table - Sections with Subsections
+
+With the inclusion of sections in the table, there are two recommended patterns for created nested collapsible sections. First is collapsible sections and static subsections, and the second is static sections and collapsible subsections. The guidance from UX is to not have both collapsible sections and collapsible subsections in a table.
```

Comment:
The guidance from UX is to ~not have both~ **only use** collapsible sections ~and~ **or** collapsible subsections in a table, **but not both**.

Reply:
Updated.